src/game/SaveLoadGame.cc

```diff
@@ -107,6 +107,7 @@
 #include "Video.h"
 #include "VSurface.h"
 #include "WorldDef.h"
+#include "Militia_Control.h"
```

Comment:
The includes in this file are sorted alphabetically; please keep it that way. Other than that your PR looks fine.